Realm/RLMObjectBase.mm

```diff
@@ -289,3 +269,31 @@ - (NSUInteger)hash {
 }
 
 @end
+
+
+
+Class RLMObjectUtilClass(bool isSwift) {
+    static Class objectUtilObjc = [RLMObjectUtil class];
+    static Class objectUtilSwift = NSClassFromString(@"RealmSwift.ObjectUtil");
+    return isSwift && objectUtilSwift ? objectUtilSwift : objectUtilObjc;
```

Comment:
In that case, won't things explode if you have a Swift subclass of `RLMObject` and you also have `RealmSwift`?